modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,15 @@
+# View Session
+
+This page displays all available information collected for an individual recording session.
+
+Summary information about the candidate, the recording session and the hardware used is displayed.
+
+Recording files can be downloaded individually or as a compressed folder (*All Files* download):
+
+- EEG: the file containing the session recording data.    
+- Electrode info (tsv): contains electrode locations.    
+- Channels info (tsv): channel status and filter settings.    
+- Events (tsv): events recorded during the session.   
```

Comment:
```suggestion
- Events (tsv): time-ordered events during the session.   
```
"events recorded" -- that's not exactly what a BIDS Events.tsv does, given the recording is the signal.  Can you find a better word or two from the BIDS spec?

Reply:
The original paper mentioned the following when describing the events files:

> [...] events (both stimuli and responses) recorded during the experiments.

[[1]](https://www.nature.com/articles/sdata201644#MOESM149) Step 3 under "Required, recommended and optional metadata".

Let me know what you think. If it's best to keep it "time-ordered", I'll commit the change.

Reply:
yeah, it's just a bit ambiguous.  i agree, let's keep the full wording:
```suggestion
- Events (tsv): events (both stimuli and responses) recorded during the session.   
```